elements/pfe-content-set/demo/index.html

```diff
@@ -2,219 +2,225 @@
-          </pfe-content-set-group>
-          <pfe-content-set-group>
-            <h2 pfe-heading>Heading 2</h2>
-            <p>Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua</p>
-            <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-          </pfe-content-set-group>
-          <pfe-content-set-group>
-            <h2 pfe-heading>Heading 3</h2>
-            <p>At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.et clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, At accusam aliquyam diam diam dolore dolores duo eirmod eos erat, et nonumy sed tempor et et invidunt justo labore Stet clita ea et gubergren, kasd magna no rebum. sanctus sea sed takimata ut vero voluptua. est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.</p>
-            <p>Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua</p>
-          </pfe-content-set-group>
+        <h2 pfe-content-set--header>Heading 1</h2>
+        <div pfe-content-set--panel>
+          <p>Content for heading 1.</p>
+          <div class="cta">
+            <a href="https://redhat.com">Primary CTA</a>
+          </div>
+        </div>
+        <h2 pfe-content-set--header>Heading 2</h2>
+        <div pfe-content-set--panel>Content for heading 2.</div>
+        <h2 pfe-content-set--header>Heading 3</h2>
+        <div pfe-content-set--panel>Content for heading 3.</div>
       </pfe-content-set>
     </div>
+
```

Comment:
Can we remove this commented out code?

Reply:
@castastrophe I'm not seeing any commented code in this file. Perhaps it has already been removed?